src/types/pathways-model.d.ts

```diff
@@ -26,6 +26,8 @@ declare module 'pathways-model' {
   export interface State {
     label: string;
     transitions: Transition[];
+    // TODO: in PATHWAYS-256 remove this property
+    nodeType?: string;
```

Comment:
I don't think this needs to wait for PATHWAYS-256 to be pulled out does it?

Reply:
I am using this property in sidebar to determine which "type" of sidebar to display